TombEngine/Objects/TR2/Entity/tr2_dragon.cpp

```diff
@@ -234,6 +236,54 @@ namespace TEN::Entities::Creatures::TR2
 		ItemPushItem(item, laraItem, coll, 1, 0);
 	}
 
+
+	static void TriggerFireBreath(ItemInfo* item, const BiteInfo& bite, const Vector3i& speed, ItemInfo* enemy)
```

Comment:
This need to be changed to include the dragon joint rotation and allow him to attack lara on the left and right.